docs/development.md

```
@@ -1,63 +1,171 @@
+```bash
+# Use a specific namespace and tag
+QUAY_USER=username NAMESPACE=eda DEV_TAG=mytag make up
+
+# Use docker instead of podman
+CONTAINER_TOOL=docker QUAY_USER=username make up
+
+# Build for a specific platform (e.g., when on ARM building for x86)
+PLATFORM=linux/amd64 QUAY_USER=username make up
+
+# Deploy without building (use an existing image)
+BUILD_IMAGE=false DEV_IMG=quay.io/myuser/eda-server-operator:latest make up
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)